blobstore-s3/src/lib.rs

```diff
@@ -200,7 +201,13 @@ impl StorageClient {
                         object_id: cobj.object_id.clone(),
                     },
                 )
-                .await?;
+                .await?,
+            ) else {
+                return Err(RpcError::InvalidParameter(
```

Comment:
`send_chunk` is already returning an `RpcError::Rpc`. Should we simply return that instead? I don't think this error message is accurate

Reply:
It may or may not return an `RpcError::Rpc` if the sent chunks is 0, right? E.g. operation to the actor is successful but they return a response with 0 bytes

Reply:
Hmm, I might have misunderstood. So we're still early returning if `send_chunk` returns an error, and all you're adding here is a different error case to handle when `send_chunk` returns 0?

Reply:
Got it, thanks 👍